go-cosmwasm/src/error/rust.rs

```diff
@@ -35,6 +35,12 @@ pub enum Error {
         #[cfg(feature = "backtraces")]
         backtrace: snafu::Backtrace,
     },
+    #[snafu(display("Error calling the enclave: {}", msg))]
```

Comment:
This isn't an error _calling_ the enclave specifically. Also if this is going to be printed to the log we can probably just print `"{}", msg` since it will be an informative error

Reply:
Also this is the 3rd variant of the name `EnclaveErr` in the project. Maybe name this something else? (sgx-vm `EnclaveErr` and `EnclaveError` in ffi/types)

Reply:
Ok, so just writing is as `"{}", msg` is probably alright.
I was debating creating another variant `SdkError` for cases of `sgx_status_t` like i did in `VmError`, for which this message is accurate, and then change the message on this variant to something like "Error returned from the enclave", but i felt that might be excessive.
As for the naming, the name shows up in a lot of contexts, but that's because different contexts use different error types, and all of them needed a new variant for errors that come from the enclave. I agree that the naming duplication isn't great, but for each of those contexts separately i felt like that was the most appropriate name.
In any case, if you mix them up, the compiler will let you know.

Reply:
So why not call it GoEnclaveErr or something? Duplicate names will cause all sorts of fun when trying to use find and replace, or find in path (not to mention the potential `Expected EnclaveErr but got EnclaveErr`  good times)

Reply:
It's not even in Go yet :sweat_smile: 
But we can name it `GoCwEnclaveError` or something to signal the layer it's in.

Reply:
יאללה קניתי

Reply:
![image](https://user-images.githubusercontent.com/22766533/87430551-0b423c80-c5ee-11ea-8005-e432b4a52e5c.png)